_search-plugins/search-relevance/compare-search-results.md

```diff
@@ -98,6 +97,26 @@ Setting `size` to a high value (for example, larger than 250 documents) may degr
 You cannot save a given comparison for future use, so Compare Search Results is not suitable for systematic testing.
 {: .note}
 
+## Comparing OpenSearch search results using Search Relevance Workbench
+
+Using the opt-in feature Search Relevance Workbench gives you richer visualization options to see the change between two different queries.
```

Comment:
We don't need to use "opt-in" on this page. We can link to the search relevance workbench page, which has installation instructions.